test/unit/mocked_data/test_get_environment/normal/get-power-usage-information-detail.xml

```diff
@@ -0,0 +1,4 @@
+<power-usage-information>
```

Comment:
@ckishimo if you add this file here, it means a device that does not have this capability, for example your qfx5110-48s-4c, does not error when retrieving this RPC & returns exactly this XML back. Is this correct?

Reply:
@mirceaulinic not sure I'm following... the qfx5110 does not have this capability but it does not error as the `get()` is within the `try...except` clause. The two files I committed are just to pass the failing tests... The qfx5110 will indeed fail when retrieving the RPC:
```jnpr.junos.exception.RpcError: RpcError(severity: error, bad_element: None, message: command is not valid on the qfx5110-48s-4c)```
I don't have other devices (ie: virtual chassis) to test this more extensively... :cry:

Reply:
Yeah, agreed there's no good way to reproduce a RpcError using mock files.